assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -142,13 +147,21 @@ cmdline_realtime=+nohz_full=${isolated_cores} tsc=nowatchdog nosoftlockup nmi_wa
 {{end}}
 
 {{if .HighPowerConsumption}}
-cmdline_power_performance=+processor.max_cstate=1 intel_idle.max_cstate=0 intel_pstate=disable
+cmdline_power_performance=+processor.max_cstate=1 intel_idle.max_cstate=0 
 {{end}}
 
 {{if and .HighPowerConsumption .RealTime}}
 cmdline_idle_poll=+idle=poll
 {{end}}
 
+{{if .PerPodPowerManagement}}
+cmdline_pstate=+intel_pstate=passive
+{{end}}
+
+{{if and (not .PerPodPowerManagement) .HighPowerConsumption}}
```

Comment:
The template should contain as little logic as possible. I would personally make sure high power consumption and perpodpowermanagement are exclusive in the first place and move the logic to the operator.

Reply:
Well, there a a bunch of if in the template, it could be a good idea do a separate PR to move al the logic to the tuned.go. Agree?

Reply:
Move it to tuned, I think it is less clear but...

Reply:
Move again from tuned to here. There is a parsers for the template that fails:
`configparser.DuplicateOptionError: While reading from '/etc/tuned/openshift-node-performance-manual/tuned.conf' [line 162]: option 'cmdline_pstate' in section 'bootloader' already exists
 `

Reply:
Sorry I missunestood the issue. Now is fixing, the logic has been moved to tuned.go and both hints can not set to enabled at the same time.

Reply:
Now:
```
{{if .PerPodPowerManagement}}
cmdline_pstate=+intel_pstate=passive
marioferh marked this conversation as resolved.
{{end}}

{{if .HighPowerConsumption}}
cmdline_pstate=+intel_pstate=disable
{{end}}
```
I do not mark as resolve to let you see the changes.